BPA/Ch.03/GLM_Poisson.stan

```diff
@@ -0,0 +1,38 @@
+data {
+  int<lower=0> n;       // Number of years
+  int<lower=0> C[n];    // Count
+  vector[n] year;       // Year
+}
+
+parameters {
+  real alpha;
+  real beta1;
+  real beta2;
+  real beta3;
+}
+
+transformed parameters {
+  vector[n] log_lambda;
+
+  log_lambda <- alpha +
+                beta1 * year +
+                beta2 * year .* year +
+                beta3 * year .* year .* year;
+}
+
+model {
+  // Priors
+  alpha ~ uniform(-20, 20);
```

Comment:
When parameters get uniform priors, they should be declared with matching constraints.  The rule is that every parameter value matching the declared constraints should have a finite log density (not -infinity).  So you need `<lower=-20,upper=20>` on `alpha`.